### DISCUSSION TITLE
Pitch sesitivity

### DISCUSSION THREAD

**Discussion by andrejetcat** [flybywiresim/aircraft#6508] [Q&A] [answered]
hello together, 

is it just my feeling, or does the Bus react mor sensitive on the pitch axis since today? 
(exp. version)

I do feel a much more agressive rotation with the same pitch input and always the same CG´s and load.

UPDATE.
On todays approach and the following landing, the plane was so sensitive, like never before. 
again, same CG´s and load AND no changes to controller settings mit linear sensitivity for the stick, as recommenced in the FFBW documentary!

**Comment by Tibu73-ui** [comment]
Same here. Too much pitch sensitivity with last exp update.

**Comment by aguther** [comment, marked as answer]
As I've written in the discussion thread for experimental (below support channel on Discord) it's true that the pitch rate was adapted. This happened based on experience in a FFS which turned out that on slow speeds far too less pitch rate was achieved. For the Thrustmaster Warthog I recommend now a sensitivity of +/- 30% for the pitch axis. Roll is unchanged.
